beanie/odm/queries/find.py

```diff
@@ -352,14 +352,14 @@ def sort(
         self,
         *args: Optional[
             Union[
-                str, Tuple[str, SortDirection], List[Tuple[str, SortDirection]]
+                Any, Tuple[Any, SortDirection], List[Tuple[Any, SortDirection]]
             ]
         ],
     ) -> "FindMany[FindQueryResultType]":
         """
         Add sort parameters
-        :param args: Union[str, Tuple[str, SortDirection],
-        List[Tuple[str, SortDirection]]] - A key or a tuple (key, direction)
+        :param args: Union[Any, Tuple[Any, SortDirection],
```

Comment:
I Any a good option? Probably it is possible to limit the number of possible types? Or there is something that blocks it?